appium/webdriver/webdriver.py

```diff
@@ -241,6 +241,7 @@ def _merge_capabilities(self, capabilities: Dict) -> Dict[str, Any]:
         """Manage capabilities whether W3C format or MJSONWP format
         """
         if _FORCE_MJSONWP in capabilities:
+            logger.warning("[Deprecated] 'forceMjsonwp' capability will be dropped")
```

Comment:
I would rather add a more detailed explanation of why the capability is going to be dropped and what is necessary to do in order to be ready for it